README.md

```diff
@@ -222,10 +222,18 @@ $ dtmate diff "11:00AM" "11:00PM"
 $ dtmate diff 2024-06-07T08:00:00Z 2024-06-08T09:02:03Z
 1 day 1 hour 2 minutes 3 seconds
 
+# same input, also convert to seconds only, brief format
+$ dtmate diff 2024-06-07T08:00:00Z 2024-06-08T09:02:03Z --conv s -b
+90123s
+
 # using timezone offset
 $ dtmate diff 2024-06-07T08:00:00Z 2024-06-07T08:05:05-05:00
 5 hours 5 minutes 5 seconds
 
+# same input, also convert duration to minutes and seconds
+$ dtmate diff 2024-06-07T08:00:00Z 2024-06-07T08:05:05-05:00 -c ms
```

Comment:
I'm surprised here the ms, I would have expected a result in milliseconds.

Here result and comment are talking about minutes and seconds.

But then, how to format in milliseconds?

Reply:
This problem is solved  by using a `.` to differentiate sub-second durations.  This is also documented when running `dtmate --help`:

* `Separate sub-second brief units with a dot`

Examples:
```shell
$ diff now "2020-01-01 11:12:13.123456789" -c ms.msusns
-2566445 minutes 40 seconds 876 milliseconds 542 microseconds 985 nanoseconds

$ go run cmd/dtmate/main.go diff now "2020-01-01 11:12:13.123456789" -c WDhms.ms
-254 weeks 4 days 6 hours 6 minutes 10 seconds 876 milliseconds
```